### PR TITLE
fix(ssr): handle undefined/null element attribute values

### PR DESCRIPTION
**Pull request by daKmoR**
closes https://github.com/lit/lit/issues/2572

### CODE REVIEW

File: packages/labs/ssr/src/test/lib/render-lit_test.ts [hunk 1]
```diff
@@ -218,6 +218,36 @@ test('element with property', async () => {
   );
 });
 
+test('element with attribute', async () => {
```
Comment (aomarks) [inline packages/labs/ssr/src/test/lib/render-lit_test.ts:221]: Could you add a case for `'element with empty string attribute'` too?

**Review by aomarks — APPROVED** [review]
Looks great! Thanks for the PR. Just one small request to add a quick extra test case as well.

File: packages/labs/ssr/src/test/lib/render-lit_test.ts (hunk 1 above)
Comment (daKmoR) [reply packages/labs/ssr/src/test/lib/render-lit_test.ts:221]: added 👍

File: packages/labs/ssr/src/test/lib/render-lit_test.ts [hunk 2]
```diff
@@ -218,6 +218,46 @@ test('element with property', async () => {
...
```
Comment (justinfagnani) [inline packages/labs/ssr/src/test/lib/render-lit_test.ts:221]: These tests should be added here: https://github.com/lit/lit/blob/main/packages/labs/ssr/src/test/integration/tests/basic.ts#L883

This way we get server rendering with an assertion, then client hydration that checks that there's no mutations.

**Review by justinfagnani — CHANGES_REQUESTED** [review]

File: packages/labs/ssr/src/test/lib/render-lit_test.ts (hunk 2 above)
Comment (daKmoR) [reply packages/labs/ssr/src/test/lib/render-lit_test.ts:221]: for "normal" rendering it's already there [AttributePart accepts undefined](https://github.com/lit/lit/blob/main/packages/labs/ssr/src/test/integration/tests/basic.ts#L940) and that worked before already

it's about the rendering of attributes of an element... so I added it to `LitElement: Attribute binding`

**Review by aomarks — APPROVED** [review]

**Comment by daKmoR** [comment]
this bug is quite annoying - anything still blocking this fix? 🤔

**Pull request merged**